lib/elixir/lib/inspect.ex

```diff
@@ -302,7 +302,12 @@ end
 
 defimpl Inspect, for: Map do
   def inspect(map, opts) do
-    list = Map.to_list(map)
+    list =
+      if Keyword.get(opts.custom_options, :sort_maps) do
+        map |> Map.to_list() |> :lists.sort()
```

Comment:
I benched this with the new `:maps.iterator(map, :ordered)`: https://github.com/sabiwara/elixir_benches/commit/71055e9bad6e47272a8fad1e918ccb979281c21d, but this implementation has the benefit of:
- being faster or equivalent
- using half as much memory
- work on OTP < 26

So I went with this one instead.